modules/led.sh

```diff
@@ -528,6 +541,43 @@ function easter {
   led green "$current_green"
 }
 
+function lantern{
+    currentGreen = $(led "green")
+    currentRed = $(led "red")
```

Comment:
When assigning to a variable, you have can't have spaces between the variable, the equals sign, and the value.

So `currentGreen = $(led "green")` should be `currentGreen=$(led "green")`. Same for `currentRed`.

Also, it would be great if you could keep the variable names the same as the other functions. So `current_green` and `current_red`.